### ISSUE TITLE
PHP: .dll build error on Windows for 1.11.0RC1

### ISSUE THREAD

**Issue by stanley-cheung** [grpc/grpc#14960]
Error:
```
handshaker.pb.c
ext\grpc\src\core\tsi\alts\handshaker\handshaker.pb.c(68): warning C4305: 'initializing': truncation from 'std::size_t' to 'pb_size_t'
ext\grpc\src\core\tsi\alts\handshaker\handshaker.pb.c(119): error C2118: negative subscript
transport_security_common.pb.c
transport_security_common_api.cc
ext\grpc\src\core\tsi\alts\handshaker\alts_tsi_handshaker.cc(310): warning C4800: 'int': forcing value to bool 'true' or 'false' (performance warning)
NMAKE : fatal error U1077: '"C:\Program Files (x86)\Microsoft Visual Studio 14.0\VC\BIN\amd64\cl.exe"' : return code '0x2'
Stop.
```

This is PHP 7.0 x vc14 (Visual Studio 2015)

To see the log file:
 - https://windows.php.net/downloads/pecl/releases/grpc/1.11.0rc1/logs/
 - Download `php_grpc-1.11.0rc1-7.0-nts-vc14-x64-logs.zip`
 - Unzip
 - Open `make-php_grpc-1.11.0rc1-7.0-nts-vc14-x64.txt`

PECL pre-built .dll for us every time we upload a release. All the recent previous releases are fine. This is a new issue introduced in 1.11.0.

See previous PECL releases: https://pecl.php.net/package/grpc

Looks like a C core issue.

**Comment by stanley-cheung** [comment]
Similar error for PHP 7.2 x VS 2017:

Download `php_grpc-1.11.0rc1-7.2-nts-vc15-x64-logs.zip`

Error:

```
handshaker.pb.c
ext\grpc\src\core\tsi\alts\handshaker\handshaker.pb.c(68): warning C4305: 'initializing': truncation from '::size_t' to 'pb_size_t'
ext\grpc\src\core\tsi\alts\handshaker\handshaker.pb.c(119): error C2118: negative subscript
transport_security_common.pb.c
transport_security_common_api.cc
NMAKE : fatal error U1077: '"C:\Program Files (x86)\Microsoft Visual Studio\2017\Enterprise\VC\Tools\MSVC\14.13.26128\bin\HostX64\x64\cl.exe"' : return code '0x2'
Stop.
```

**Comment by stanley-cheung** [comment]
Related to #14760 and #14761?

**Comment by ZhouyihaiDing** [comment]
#14825 and #14807 solved the problem on macOS and Linux.
I am trying to figure out how to install gRPC-PHP and test it in Windows by command line.

**Comment by stanley-cheung** [comment]
Issue fixed at 1.11.0RC2
